### PR TITLE
Allow comments in tsconfig and show user error message if load fails

### PR DESCRIPTION
**Pull request by sazzy4o**
Summary:
-Update [testyts](https://github.com/Testy/TestyTs) dependency
-Allow comments in tsconfig (via [jsonc-parser](https://github.com/microsoft/node-jsonc-parser))
-Show user error message if load fails
-Fix align in README
-Add tsc task to launch (so you don't need to run tsc manually every time you debug)

Now it will show error if load fails because of tsc error or bad config file:
image

Let me know if there are any changes you want me to make

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -43,9 +43,10 @@ export class TestyTsAdapter implements TestAdapter {
         fork(require.resolve('./workers/testLoader.worker.js'), [],
             { cwd: this.workspace.uri.fsPath, execPath: this.config.nodePath, execArgv: [] })
             .on('message', (response: TestSuiteInfo) => {
-                if (response instanceof String) {
+                if (response instanceof String || typeof response === "string") {
```
Comment (sazzy4o) [inline src/adapter.ts:46]: The original logic does not work as you would expect see:
https://stackoverflow.com/questions/203739/why-does-instanceof-return-false-for-some-literals

File: src/workers/testRunner.worker.ts [hunk 2]
```diff
@@ -18,14 +20,15 @@ catch (err) {
...
+
+    const testyConfig: TestyConfig = parse(readFileSync(resolve(process.cwd(), 'testy.json'),{encoding:'utf8'}));
+    const tsconfig = parse(readFileSync(resolve(process.cwd(), testyConfig.tsconfig || 'tsconfig.json'),{encoding:'utf8'}));
 
     const tests = await testLoader.loadTests(process.cwd(), testyConfig.include, tsconfig);
     testsIds = await tests.accept(new TestFinderVisitor(testsIds));
     process.send(<TestRunStartedEvent>{ type: 'started', tests: testsIds });
 
-    let runner: TestVisitor<Report> = new TestRunnerVisitor();
+    let runner: TestVisitor<Report> = new TestRunnerVisitor(process);
```
Comment (sazzy4o) [inline src/workers/testRunner.worker.ts:28]: Required for `testyts` 1.0.1

**Comment by Aboisier** [comment]
Thanks for the contribution!! I'll review/merge/deploy this as soon as possible.

**Comment by danielmeza** [comment]
Nice job!!

File: package.json [hunk 3]
```diff
@@ -33,16 +33,17 @@
         "publish": "vsce publish"
     },
     "dependencies": {
-        "testyts": "0.7.x",
+        "jsonc-parser": "^2.2.1",
```
Comment (Aboisier) [inline package.json]: Typescript comes with a "readConfigFile" method, I'd rather use it than add an external dependency.

**Review by Aboisier — APPROVED** [review]
I made a few changes (got rid of jsonc and used build-in method instead, and refactored the configs-loading stuff). Thanks a lot for the contribution! 💯

**Pull request merged**